example/package.json

```diff
@@ -11,9 +11,9 @@
     "build:ios": "react-native build-ios --scheme KlaviyoReactNativeSdkExample --mode Debug --extra-params \"-sdk iphonesimulator CC=clang CPLUSPLUS=clang++ LD=clang LDPLUSPLUS=clang++ GCC_OPTIMIZATION_LEVEL=0 GCC_PRECOMPILE_PREFIX_HEADER=YES ASSETCATALOG_COMPILER_OPTIMIZATION=time DEBUG_INFORMATION_FORMAT=dwarf COMPILER_INDEX_STORE_ENABLE=NO\""
   },
   "dependencies": {
-    "@react-native/new-app-screen": "0.80.2",
+    "@react-native/new-app-screen": "0.81.0",
     "react": "19.1.0",
-    "react-native": "0.80.2"
+    "react-native": "0.81.0"
```

Comment:
Shall we use the latest patch? [0.81.5](https://github.com/facebook/react-native/releases/tag/v0.81.5)